modules/appeals_api/app/swagger/contestable_issues/v0/swagger.json

```diff
@@ -548,10 +548,6 @@
           }
         }
       },
-      "non_blank_string": {
```

Comment:
This should still be included. Seems like some shared schemas are missing w/ the change to `DocHelpers.decision_reviews?`.